pkg/build/builder/util.go

```diff
@@ -74,26 +74,40 @@ func GetCGroupLimits() (*s2iapi.CGroupLimits, error) {
 	}
 
 	// different docker versions seem to use different cgroup directories,
-	// check for both.
-	cgroupDir := "/sys/fs/cgroup/cpuacct,cpu"
+	// check for all of them.
+
+	// seen on rhel systems
+	cpuDir := "/sys/fs/cgroup/cpuacct,cpu"
+
+	// seen on fedora systems with docker 1.9
+	// note that in this case there is also a /sys/fs/cgroup/cpu that symlinks
+	// to /sys/fs/cgroup/cpu,cpuacct, so technically the next check
+	// would be sufficient, but it seems better to rely on the real directory
+	// rather than a symlink.
 	if _, err := os.Stat("/sys/fs/cgroup/cpu,cpuacct"); err == nil {
-		cgroupDir = "/sys/fs/cgroup/cpu,cpuacct"
+		cpuDir = "/sys/fs/cgroup/cpu,cpuacct"
 	}
 
-	cpuQuota, err := readInt64(filepath.Join(cgroupDir, "cpu.cfs_quota_us"))
+	// seen on docker1.10 fedora systems
```

Comment:
so I installed 1.10 on fedora and it does have cpu,cpuacct. So actually this is seen in debian-like distros like TinyCore linux in boot2docker.